library/ctr_drbg.c

```diff
@@ -103,14 +99,12 @@ void mbedtls_ctr_drbg_free( mbedtls_ctr_drbg_context *ctx )
         return;
 
 #if defined(MBEDTLS_THREADING_C)
-    mbedtls_mutex_free( &ctx->mutex );
+    if( ctx->f_entropy != NULL )
```

Comment:
As the PR description says, some comments should be added. I understand this is a check on whether the context has been seeded, but out of the context of that PR, that wouldn't be obvious.

Reply:
I added some comments.